src/Writer/BatchWriter.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Ddeboer\DataImport\Writer;
+
+use Ddeboer\DataImport\Writer;
+
+/**
+ * @author Markus Bachmann <markus.bachmann@bachi.biz>
+ */
+class BatchWriter implements Writer
+{
+    private $delegate;
+
+    private $size;
+
+    private $queue;
+
+    public function __construct(Writer $delegate, $size = 20)
+    {
+        $this->delegate = $delegate;
+        $this->size = $size;
+    }
+
+    public function prepare()
```

Comment:
By definition, writers should return self in all methods. But I :+1: removing fluent interfaces.